src/components/LinterMessage/index.tsx

```diff
@@ -58,18 +58,31 @@ const renderDescription = (description: LinterMessage['description']) => {
   }, []);
 };
 
-type PublicProps = {
+export type PublicProps = {
+  highlight?: boolean;
```

Comment:
Instead of receiving this prop, it would be simpler to wrap this component in `withRouter()`, parse the `props.location` query string, and compare `messageUid` to `message.uid`. It pushes the logic into `LinterMessage` but I think that's fine. The query string is global so it makes sense to access it anywhere. What do you think?

Reply:
Yes, that seems like a better approach. Thanks!